notes/2024/2024-04-04.md

```diff
@@ -0,0 +1,83 @@
+# 2024-04-04ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2024-03-21-transcript.md`](2024-03-21-transcript.md)
+
+## Attending
+
+- Nicholas C. Zakas (@nzakas) - TSC
+- Milos Djermanovic (@mdjermanovic) - TSC
+- Francesco Trotta (@fasttime) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### Status of previous actionable items
+
+- [x] [feat: support the new flat config](https://github.com/eslint/create-config/issues/51) (Completed)
+- [x] [Switch Playground to flat config](https://github.com/eslint/eslint.org/issues/507) (Completed)
+
+### [Change Request: `--inspect-config` CLI option](https://github.com/eslint/eslint/issues/18255)
+
+**TSC Summary:** This issue seeks to add a --inspect-config CLI option that will launch the config inspector (similar to --init does with @eslint/create-config).
+
+**TSC Question:** Shall we accept this proposal?
+
+**Resolution:** We've agreed to accept this proposal
+
+**Action Items:** @nzakas will work on the PR
+
+### [feat: Support different names for code block languages](https://github.com/eslint/eslint-plugin-markdown/issues/245)
+
+**TSC Summary:** This change seeks to use recognizable filename extensions for common code block types like node, javascript, ecmascript to .js, markdown to .md, and typescript to .ts. Currently, the plugin would use the code block type as the filename extension, so javascript becomes 0.javascript, which is a bit confusing.
+
+**TSC Question:** This represents a breaking change, do we want to accept this and publish a new major version?
+
+**Resolution:** We've agreed to accept this proposal but will omit `node` option at this time.
+
+### [Change Request: Provide a way for rules to apply suggestions to other files](https://github.com/eslint/eslint/issues/17881)
```

Comment:
Fyi, I've removed the `tsc-agenda` label but maybe i shouldn't have since we table this conversation. Please let me know if I should add it back so we don't lose track of this issue. 👍

Reply:
We are going to try to handle this directly on the issue, so no need for the label at this point.